grunt-configs/karma.js

```diff
@@ -6,7 +6,7 @@ module.exports = function(grunt, options) {
             singleRun: options.singleRun,
             browserDisconnectTimeout: 10000,
             browserDisconnectTolerance: 3,
-            browserNoActivityTimeout: 60000,
+            browserNoActivityTimeout: 400000,
```

Comment:
This was because locally PhantomJS takes longer to start up (I wasn't able to understand why this is the case, doesn't seem to be an issue when running in TeamCity)